File: SOLUTIONS/reverse-word.js
function reverse_word(input) {
    let words = input.split(' ');
    let reverse_word = words.reverse();
    let word = reverse_word[0];
    for (let index = 1; index < reverse_word.length; index++) {
        word = word + ' ' + reverse_word[index];        
    }
    console.log(word); 
}

//reverse_word('My name is Michele')
//Michele is name My

File: SOLUTIONS/reverse_word.py
from __future__ import print_function
def reverseWords(input): 
      
    # split words of string separated by space 
    inputWords = input.split(" ") 
  
    # reverse list of words 
    # suppose we have list of elements list = [1,2,3,4],  
    # list[0]=1, list[1]=2 and index -1 represents 
    # the last element list[-1]=4 ( equivalent to list[3]=4 ) 
    # So, inputWords[-1::-1] here we have three arguments 
    # first is -1 that means start from last element 
    # second argument is empty that means move to end of list 
    # third arguments is difference of steps 
    inputWords=inputWords[-1::-1] 
  
    # join words with space 
    output = ' '.join(inputWords) 
      
    return output 
  
if __name__ == "__main__": 
    input = 'geeks quiz practice code'
    print(reverseWords(input)) 
